client/game/board/Board.svelte

```diff
@@ -2,29 +2,44 @@
 	export let cards = [];
 
 	import Card from "./Card.svelte";
-	import { createEventDispatcher } from "svelte";
+	import { socket } from "../../connectivity.js";
 	import { areCardsEqual, arrayContainsCard, isValidPlay } from "../shared.js";
 
-	const dispatch = createEventDispatcher();
+	const keyboardMap = ["qwertyu", "asdfghj", "zxcvbnm"].map(line => line.split(""));
+
+	function getKey(i) {
+		if (i >= keyboardMap.length * keyboardMap[0].length) {
+			return "";
+		} else {
+			return keyboardMap[i % 3][parseInt(i / 3)].toUpperCase();
+		}
+	}
 
 	let selection = [];
 
 	function cardClicked(e) {
+		if (selection.length >= 3) {
```

Comment:
Can we avoid using magic numbers and give this number a name?

It's supposed to be the number of cards in a play, so how about `NUM_CARDS_FOR_PLAY`?

Reply:
Implemented